source/acquaman/AMGenericContinuousScanController.cpp

```diff
@@ -101,6 +101,9 @@ void AMGenericContinuousScanController::buildScanControllerImplementation()
 
 		foreach (AMRegionOfInterest *region, configuration_->regionsOfInterest()){
 
+			// This is sufficient to add a region of interest on all detectors as they should by synchronized via AMBeamline::synchronizeXRFDetectors.
+			detector->addRegionOfInterest(region);
```

Comment:
@helfrij, there is a `detector->removeAllRegionsOfInterest()` on line 87 of this file so there shouldn't be an issue with double adding.  I believe @XuetaoLiu-CLS just moved this method.